packages/semantic-ui/src/CheckboxesWidget/CheckboxesWidget.js

```diff
@@ -16,19 +16,20 @@ function deselectValue(value, selected) {
   return selected.filter(v => v !== value);
 }
 
-function CheckboxesWidget({
-  id,
-  disabled,
-  options,
-  value,
-  autofocus,
-  readonly,
-  onChange,
-  onBlur,
-  onFocus,
-  formContext,
-  schema,
-}) {
+function CheckboxesWidget(props) {
+  const {
```

Comment:
Let's undo this change, unless you think it's better to have it this way?

Reply:
i prefer this approach, as the other themes do it this way as well, so they can perform functions on all props, or validate props and so forth